### DISCUSSION TITLE
莫大的讽刺

### DISCUSSION THREAD

**Discussion by xinkecn** [WeNeedHome/SummaryOfLoanSuspension#299] [Show and tell]
人民民主专政的以工农为主体的社会主义国家，居然怕被人民讨论。莫大的讽刺

**Comment by GeraldTao** [comment]
人民的定义：人民是对一个经济体认同并有归属感的人群集合，是区别于敌人的政治概念。

家都没了，还有归属感吗？没有归属感，那就不是人民啦~

**Comment by ruo-xi** [comment]
宪法很灵活的，以前还是无产阶级专政呢，直接给你把专政的人写明 写死了，看看现在 人民民主专政 不就是个笑话吗，老百姓也配叫人民，快去996去，当特色社会（资本主义社会）的燃料。 唉 悲哀。
